### ISSUE TITLE
Arabic Word / Quran verses not well displayed

### ISSUE THREAD

**Issue by Gujarats** [quran/quran.com-frontend#787]
Sometimes when scrolling down after reading some verses I often got the verses of the quran is not well displayed as for example like this : 

![bugs](https://cloud.githubusercontent.com/assets/10476678/25848418/794dbe50-34e4-11e7-8609-d2af13e873ff.png)

Is there any way to check if the quran verses is well displayed? like for example when loading the verses when scrolling down?

my assumption is this issue happened because of the internet connection sometimes not stable so that all the images or data is not received when downloading.

**Comment by naveed-ahmad** [comment]
Yeah fonts failed to load due to slow connection. InshAllah new fonts will resolve this break.

**Comment by Gujarats** [comment]
Are you guys using http2.0 to get all the resources like images , script and stuff? 

Correct me if I'm wrong here but maybe it will give the boost improvement for the loading speed.

**Comment by mmahalwy** [comment]
@naveed-ahmad we should try to do the text output until fonts load as previously, or work on the fonts again please :)

**Comment by naveed-ahmad** [comment]
Trying to complete FB app and fix the fonts before Ramdan, both task are complicated :(
